docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,125 @@
+such as which sources are collected by default.
+
+// TODO: Update links when distro docs are markdown-ified
+Get started with an Elastic Distribution for OpenTelemetry Language SDK:
+
+* preview:["The Elastic Distribution for OpenTelemetry Java is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-java[Java]
+* preview:["The Elastic Distribution for OpenTelemetry .NET is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-dotnet[.NET]
+* preview:["The Elastic Distribution for OpenTelemetry Node.js is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-node[Node.js]
+
+[NOTE]
+====
+For more details about OpenTelemetry distributions in general,
+visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+====
+
+[float]
+[[apm-otel-api-sdk-elastic-agent]]
+== OpenTelemetry API/SDK + Elastic APM agent
+
+Use the OpenTelemetry API/SDKs with Elastic APM agents to translate OpenTelemetry API calls to Elastic APM API calls.
+This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--avoiding
+vendor lock-in and having to redo manual instrumentation.
+
+// Do we want to say anything about why you wouldn't choose this
+// method to send data to Elastic?
 
-To unlock the full power of the {stack}, use the OpenTelemetry API/SDKs with Elastic APM agents,
-currently supported by the Java, Python, .NET, and Node.js agents.
-These Elastic APM agents translate OpenTelemetry API calls to Elastic APM API calls.
-This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--
-avoiding vendor lock-in and having to redo manual instrumentation.
+This is currently supported by the Java, Python, .NET, and Node.js agents.
 
-<<apm-open-telemetry-with-elastic,Get started>>.
+<<apm-open-telemetry-with-elastic,*Get started with an OpenTelemetry API/SDK + an Elastic APM agent →*>>
 
-**OpenTelemetry agent**
+image::images/apm-otel-api-sdk-elastic-agent.png[]
+
+{nbsp} +
+
+[float]
+[[apm-otel-api-sdk]]
+== OpenTelemetry API/SDK
 
 The {stack} natively supports the OpenTelemetry protocol (OTLP).
 This means trace data and metrics collected from your applications and infrastructure by an
 OpenTelemetry agent can be sent directly to the {stack}.
 
-<<apm-instrument-apps-otel,Get started>>.
+// Do we want to say anything about why you would/wouldn't choose this
+// method to send data to Elastic?
```

Comment:
This path is recommended for technologies which we do not provide a solution for. For example, users who want to instrument C or C++ should use raw OTel ([opentelemetry cpp](https://github.com/open-telemetry/opentelemetry-cpp)). Other languages include erlang, lua, perl.

Downsides:
* we don't support raw otel agents
* no enterprise apm features
* performance efficiency